source/_patterns/01-molecules/tag-list/tag-list~ar.yaml

```diff
@@ -8,8 +8,12 @@ list:
           href: '#'
         text: أحمر
     - content:
+        attributes:
+          href: '#'
```

Comment:
Would suggest we leave the ones without links; though usage is meant to be with one we can't guarantee it.